opus/application/apps/tools/file_utils.py

```diff
@@ -267,6 +267,7 @@ def get_displayed_browse_products(opus_id, version_name='Current'):
     """
```

Comment:
We should say something in the comment about the return being a list of tuples (medium, full)